app/helpers/admin/user_role/dropdown.rb

```diff
@@ -75,7 +75,7 @@ def own_user?
 
       def button_enabled?
         return false if own_user?
-        if @user.admin? && !@current_user.admin?
+        if (@user.admin? && !@current_user.admin?) || @user.root?
```

Comment:
isn't it possible to check that by policy ? ask policy ?
policy(...).permission-name?